test/test_cuda.py

```diff
@@ -79,7 +79,7 @@ def make_sparse_tensor(t, n, *sizes):
         torch.cat([torch.LongTensor(1, n).random_(s) for s in sizes], 0))
     v = tensor._values()
     v = v.new(n).copy_(torch.randn(n))
-    return t(i, v, torch.Size(sizes))
+    return t(i, v, torch.Size(sizes)).coalesce()
```

Comment:
Note to our sparse colleagues (@cpuhrsch, @pearu, @IvanYashchuk): sparse tests in test_cuda.py should probably be moved into test_sparse.py